default.nix

```diff
@@ -1,22 +1,33 @@
 let
-  config = {
-    packageOverrides = super: rec {
+  thisOverlay = self: super: {
+    my-R = self.rWrapper.override {
+      packages = with self.rPackages; [ dplyr ];
+    };
```

Comment:
This is my attempt at trying to get `dplyr` working (and where I use `my-R` below), but it doesn't look like it is working.

I don't know enough about R, R-packaging in nixpkgs, and `inline-r` to figure out why this is not working.

Reply:
However, Tweag (the authors of `inline-r`) are big nix-users, so you may want to get in touch with somewhere there to ask how to do this.  I'm sure they would have some projects doing something similar to what you're trying to do.